auth/server/src/main/java/com/seagates3/dao/PolicyDAO.java

```diff
@@ -47,7 +49,9 @@ public Policy find(Account account, String name)
     public
      Policy findByArn(String arn, Account account) throws DataAccessException;
     public
-     List<Policy> findAll(Account account) throws DataAccessException;
+     List<Policy> findAll(
```

Comment:
we are changin a public API signature. If this was recently written code and we are sure we have changed all the calling instances then its fine.
If this is a old code, then we should not change the signature as others in opensource community may be using it

Reply:
@kupranay , This is recently written code. and I have changed all the calling instances.